packages/devtools_app/lib/src/screens/debugger/codeview.dart

```diff
@@ -450,10 +484,11 @@ class _CodeViewState extends State<CodeView>
     return Center(
       child: ElevatedButton(
         autofocus: true,
-        onPressed: () => widget.controller.toggleFileOpenerVisibility(true),
+        onPressed: () =>
+            widget.codeViewController.toggleFileOpenerVisibility(true),
         child: Text(
           'Open a file ($openFileKeySetDescription)',
-          style: theme.textTheme.subtitle1,
+          style: theme.textTheme.titleMedium,
```

Comment:
you're probably on a newer version of flutter locally than our bots run on. I'd checkout flutter  3.3.0-0.5.pre and see if this is still valid

Reply:
Ah yeah, good point. I saw the deprecation, figured I'd fix it without thinking.